packages/nimble-components/CONTRIBUTING.md

```diff
@@ -77,15 +77,33 @@ Before building a new component, 3 specification documents need to be created:
 
 ## Develop new components
 
-### Marking a component as in development
+### Marking a component as incubating
 
-If a component will require multiple pull requests before having a complete and stable API, it should be marked as "in-development" to indicate to clients that they shouldn't start using it yet. To do this:
+If a component is not ready for general use, it should be marked as "incubating" to indicate that status to clients. A component could be in this state if any of the following are true:
```

Comment:
I guess based on the other comments I had maybe a underlying thought that the incubating namespace is for components that are being introduced that we expect to likely not be feature complete by the end of their expected dev cycle. Things that are truly prototypes / experimental / their is agreement up-front they will not be complete.

Versus things that we fully expect to dev to completion (barring extreme unforeseen circumstance) would just start-out in the top-level namespace but have the storybook annotations, etc.

Maybe that's not a good strategy? Should all new component development start in incubating?

Reply:
And kind of related and not directly captured, are we also capturing details for how Angular and Blazor incubating components are organized?

Should Angular be under a secondary entrypoint `@ni/nimble-angular/incubating/`? What namespace for Blazor `NimbleBlazor.Incubating`, `NimbleBlazor.Tests.Incubating`, `NimbleBlazor.Tests.Acceptance.Incubating`?

Reply:
I guess this question still applies: Should all new component development start in incubating?

Reply:
And now that incubating does not require moving source location, etc, and is primarily a documentation and semver behavior the answer is probably more confidently yes

Reply:
I'm ok with all new components starting out in incubating. I couldn't think of a succinct way to make this more explicit than listing "it is still in development" below. (I'm open to suggestions to do in a follow up). I also updated the new component issue template to mention removing this status at the end of dev.